pkg/render/logstorage.go

```diff
@@ -2122,17 +2132,52 @@ func (m managedClusterLogStorage) linseedExternalRoleAndBinding() (*rbacv1.Clust
 		},
 	}
 
-	// Bind the permission to the tigera-fluentd namespace. Other controllers may also bind
+	// These permissions are necessary so that we can fetch the operator namespace of the managed cluster from the
+	// management cluster so that we're copying secrets into the right place in a multi-tenant environment.
+	configMapsRole := &rbacv1.ClusterRole{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "tigera-linseed-configmaps",
+		},
+		Rules: []rbacv1.PolicyRule{
+			{
+				APIGroups: []string{""},
+				Resources: []string{"configmaps"},
+				Verbs:     []string{"get"},
+			},
+		},
+	}
+
+	// Bind the secrets permission to the tigera-fluentd namespace. Other controllers may also bind
 	// this cluster role to their own namespace if they require linseed access tokens.
-	binding := &rbacv1.RoleBinding{
+	secretBinding := &rbacv1.RoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "tigera-linseed",
+			Namespace: fluentdName,
+		},
+		RoleRef: rbacv1.RoleRef{
+			APIGroup: "rbac.authorization.k8s.io",
+			Kind:     "ClusterRole",
+			Name:     TigeraLinseedSecretsClusterRole,
+		},
+		Subjects: []rbacv1.Subject{
+			{
+				Kind:      "ServiceAccount",
+				Name:      "tigera-linseed",
+				Namespace: ElasticsearchNamespace,
+			},
+		},
+	}
+
+	// Bind the configmaps permission to the calico-system namespace.
+	configMapBinding := &rbacv1.RoleBinding{
```

Comment:
This as well is only needed in managed clusters?

Reply:
This specific one yes, we do have an equivalent for management clusters by way of linseed.go:188